source/s3_auto_ranged_put.c

```diff
@@ -82,8 +80,6 @@ static bool s_s3_auto_ranged_put_update(
     uint32_t flags,
```

Comment:
A few lines up, in `struct aws_s3_prepare_complete_multipart_upload_job`, delete this member:
```
/* async step: skip remaining parts from input stream that were previously uploaded */
struct aws_future_void *asyncstep_skip_remaining_parts;
```